source/_pages/privacy-policy-term.md

```diff
@@ -0,0 +1,91 @@
+<!-- <style>
+    h1 {
+        font-size:30pt;
+        font-weight: bold;
+        color: #184c4e;
+    }
+
+    h2 {
+        font-size:24pt;
+        font-weight: bold;
+        color: #184c4e;
+    }
+
+    ul {
+        list-style-type: disc;
+        margin-left: 50px;
+    }   
+</style> -->
+
+# LibreSign Privacy Notice
+&nbsp;
```

Comment:
Why a blank character?
```suggestion
&nbsp;
```

Reply:
I was test how can put space in marckdown. I am change for right option of space in entired file.

Reply:
You need to do this using CSS maybe increasing the padding